frappe/core/doctype/communication/communication.py

```diff
@@ -17,6 +17,7 @@
 from frappe.core.doctype.communication.email import validate_email
 from frappe.core.doctype.communication.mixins import CommunicationEmailMixin
 from frappe.core.utils import get_parent_doc
+from frappe.email.inbox.mark_as_trash
```

Comment:
This isn't valid syntax. Please test before raising PR! :facepalm:

Reply:
A thousand apologies had done `git stash` earlier

Reply:
fixed it